modules/appeals_api/app/swagger/decision_reviews/v2/swagger_dev.json

```diff
@@ -1094,6 +1094,137 @@
         }
       }
     },
+    "/higher_level_reviews/{uuid}/download": {
+      "get": {
+        "summary": "Download a watermarked copy of a submitted Higher-Level Review",
+        "tags": [
+          "Higher-Level Reviews"
+        ],
+        "operationId": "downloadHlr",
+        "security": [
+          {
+            "apikey": [
+
+            ]
+          }
+        ],
+        "description": "Returns a watermarked copy of the submitted Higher-Level Review PDF. The PDF will only be available\nfor download after the Higher-Level Review has progressed to a 'submitted' state. The PDF will\nbecome unavailable again once personally identifying information is deleted from the server one week after the\nHigher-Level Review progresses to a 'completed' state.\n\nA PDF will not be available unless an ICN was included when the Higher-Level Review was created,\nAnd the X-VA-ICN header submitted to this endpoint is required to match the ICN originally submitted with the\nHigher-Level Review.\n",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-VA-ICN",
+            "required": false,
+            "description": "ICN of the veteran associated with the Higher-Level Review",
```

Comment:
I had been given guidance in the past that `Veteran` should always be capitalized, but if that guidance is still correct, I expect UX will catch it in review.

Reply:
I found [the place where this is documented](https://design.va.gov/content-style-guide/capitalization).